app/views/static/_signed_out_index.html.haml

```diff
@@ -1,3 +1,5 @@
+- set_meta_tags description: 'Get structured data out of the web. Code collaboration through GitHub. Run your scrapers in the cloud.'
```

Comment:
It doesn't make sense for a url to have different meta tags in different circumstances. So, this should be set on `app/views/static/index.html.haml` so that it applies whether you're logged in or not.